game-core/src/main/java/games/strategy/triplea/delegate/BaseTripleADelegate.java

```diff
@@ -26,7 +25,6 @@
    * Creates a new instance of the Delegate.
    */
   public BaseTripleADelegate() {
-    super();
   }
```

Comment:
Unless there's another constructor in this class, this one can be removed

Reply:
It's the only one - can remove it sure enough: a4fc34d